src/dbSta/src/dbSta.cc

```diff
@@ -231,7 +231,6 @@ void dbSta::unregisterStaState(dbStaState* state)
 std::unique_ptr<dbSta> dbSta::makeBlockSta(odb::dbBlock* block)
 {
   auto clone = std::make_unique<dbSta>();
-  clone->makeComponents();
```

Comment:
Where is the other call?

Reply:
It's the line right after, `clone->initVars()` :

```c++
void dbSta::initVars(Tcl_Interp* tcl_interp,
                     odb::dbDatabase* db,
                     utl::Logger* logger)
{
  db_ = db;
  db->addObserver(this);
  logger_ = logger;
  makeComponents();
  if (tcl_interp) {
    setTclInterp(tcl_interp);
  }
  db_report_->setLogger(logger);
  db_network_->init(db, logger);
  db_cbk_ = std::make_unique<dbStaCbk>(this, logger);
  buffer_use_analyser_ = std::make_unique<BufferUseAnalyser>();
}
```